src/components/contactPicker/ContactPicker.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const ContactPicker = ({
+  name,
+  onChange,
+  contacts}) => {
+  return (
+    <>
+      <select name={name} onChange={onChange}>
+        {contacts.map((contact, index) => {
+        return <option value={contact} key={index}>{contact}</option>
```

Comment:
We shouldn't use `index` as the `key`. `contact` is unique already. Generally we want to avoid using indicies as keys.

Reply:
Done